app/controllers/games_controller.rb

```diff
@@ -15,7 +16,7 @@ def new
   def create
     @game = Game.new(game_params)
     if @game.save
-      flash[:notice] = 'You have added a new game!'
+      flash[:notice] = 'you have added a new game!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.